corehq/apps/app_manager/templates/app_manager/v1/partials/build_errors.html

```diff
@@ -295,6 +295,16 @@
                                     must go in a different module).
                                 {% endblocktrans %}
                             {% endif %}
+                        {% case "practice user config error" %}
+                            {% if error.build_profile_id %}
+                                {% blocktrans with profile=app.build_profiles|getattr:error.build_profile_id message=error.message %}
+                                    Error with Practice Mobile Worker in Application Profile {{ profile.name }}.
+                                {% endblocktrans %}
+                            {% else %}
+                                {% blocktrans with message=error.message %}
+                                    Error with Practice Mobile Worker in Application Settings.
+                                {% endblocktrans %}
+                            {% endif %}
```

Comment:
both of these blocktrans blocks prepare `message` but never use it - should `message` be a part of the text or should it be removed from the `with` clause?